.github/workflows/pr-verify.yml

```diff
@@ -274,65 +267,10 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_ACTOR: ${{ github.actor }}
 
-  publish_npm_package:
```

Comment:
Only publishing on release. Java behavior is a bit different since we are using snapshot version on that and it might be fine releasing snapshot versions to GitHub but that's also questionable. Open to discuss which way makes  more sense.

Reply:
Going to use the same process as Node and only publish on release. No more snapshots!